### ISSUE TITLE
ActivationListener isn't called on execBackwards

### ISSUE THREAD

**Issue by orausch** [deeplearning4j/deeplearning4j#8319]
#### Issue Description

I want to get the calculate the loss and gradients for a batch at the same time (#8318 related) in my training loop. @AlexDBlack suggested to add an activation listener to the variable, but this doesn't seem to work. The following example does not print the expected output from the listener:
https://gist.github.com/orausch/f78035f42940e5f614c04f32cd53a271
#### Version Information
`beta5`

**Comment by orausch** [comment]
As a user I would also expect `sd.getArrForVarName("loss")` to also work, but that returns `null`.

**Comment by AlexDBlack** [comment]
Thanks for reporting this, it has been fixed here: https://github.com/KonduitAI/deeplearning4j/pull/21
It will be merged back to Eclipse master soon
